tastypie/authorization.py

```diff
@@ -263,3 +257,88 @@ def delete_detail(self, object_list, bundle):
             raise Unauthorized("You are not allowed to access that resource.")
 
         return True
+
+
+class DjangoObjectAuthorization(Authorization):
+    '''
+    Uses permission checking from ``django.contrib.auth`` to map
+    ``POST / PUT / DELETE / PATCH`` to their equivalent Django auth
+    permissions.
+
+    Both the list & detail simply check the object they're based on.
+    Object level authorization api is added since django 1.5. However,
+    it will default to no-access unless an AUTHENTICATION_BACKENDS is
+    setup to handle those checks.
+    '''
+
+    # By default, follows `ModelAdmin` "convention" to use `app.change_model`
+    # `django.contrib.auth.models.Permission` for both viewing and updating.
+    # https://docs.djangoproject.com/es/1.9/topics/auth/default/#permissions-and-authorization
+    READ_PERM_CODE = getattr(settings, 'TASTYPIE_READ_PERM_CODE', 'change')
```

Comment:
Could you just hard code this to 'change' here, instead of adding a new setting? If people really need to change it they can subclass and override READ_PERM_CODE.